src/renderer/components/FrequencyBand.tsx

```diff
@@ -53,6 +53,13 @@ import { FilterActionEnum, useAquaContext } from '../utils/AquaContext';
 import Slider from './Slider';
 import '../styles/FrequencyBand.scss';
 
+const NO_GAIN_FILTER_TYPES = new Set([
```

Comment:
Would probably be good to move this out to constants.ts, have FrequencyBandTest loop through NO_GAIN_FILTER_TYPES